todo_app/tests/test_viewmodel.py

```diff
@@ -0,0 +1,55 @@
+import pytest, requests
+from todo_app.Item import Item
+from todo_app.ViewModel import ViewModel
+
+from todo_app.data.trello_items import get_items
+
+@pytest.fixture
+def viewmodel_test():
+    items = [Item("1", "To Do Test", status="To Do"), Item("2", "Doing Test", status="Doing"), Item("3", "Done Test", status="Done")]
+    return ViewModel(items)
+
+def test_pytest_works():
+    assert(True)
+
+def test_viewmodel_doing(viewmodel_test):
+    assert len(viewmodel_test.doing) == 1
+    assert viewmodel_test.doing[0].name == "Doing Test"
+    with pytest.raises(IndexError):
+        viewmodel_test.doing[1]
+
+def test_viewmodel_done(viewmodel_test):
+    assert len(viewmodel_test.done) == 1
+    assert viewmodel_test.done[0].name == "Done Test"
+    with pytest.raises(IndexError):
+        viewmodel_test.done[1]
+
+def test_viewmodel_todo(viewmodel_test):
+    assert len(viewmodel_test.to_do) == 1
+    assert viewmodel_test.to_do[0].name == "To Do Test"
+    with pytest.raises(IndexError):
+        viewmodel_test.to_do[1]
+    
+def test_get_items(monkeypatch):
```

Comment:
This feels like it merits its own file since it's testing quite different behaviour